docs/topics/wasm-wasi.md

```diff
@@ -0,0 +1,127 @@
+[//]: # (title: Get started with Kotlin/Wasm and WASI)
+
+> Kotlin/Wasm is [Alpha](components-stability.md). It may be changed at any time.
+>
+> [Join the Kotlin/Wasm community.](https://slack-chats.kotlinlang.org/c/webassembly)
+>
+{style="note"}
+
+This tutorial demonstrates how to run a [WASI](https://wasi.dev/) application with [Kotlin/Wasm](wasm-overview.md) in IntelliJ IDEA. 
```

Comment:
We should mention here that Kotlin/Wasm currently supports "0.1" (also known as "Preview 1") version of WASI.

We can also mention that we plan to support WASI 0.2 in the future.

Reply:
the issue about supporting WASI 0.2 [KT-64568](https://youtrack.jetbrains.com/issue/KT-64568)